website/next.config.js

```diff
@@ -111,6 +111,15 @@ module.exports = phase => ({
           permanent: true,
         }
       }),
+      ...[
```

Comment:
This can be

```
{
  source: '/docs/reference/cli-reference',
  destination: '/docs/reference/cli',
  permanent: true,
}
```

Reply:
was trying to be consistent with the other redirects, but that works too. changed